### PR TITLE
CompanyPage_is-looking-for

### PR DESCRIPTION
**Pull request by olehoef**
Only localization of Master programs is missing. Should be updated once #261 is merged.

### CODE REVIEW

File: app/companyCall.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ export const API_Call_Company_Details = async(ids) => {
         const programs1 = (await response1.json()).data;
         const response1p1 = await fetch(`${Base_URL}items/companies_programs_1/?filter[id][_in]=${ids.programs.join(',')}`, {headers});
         const programs1p1 = (await response1p1.json()).data;
+        const programs1p1Ids = programs1p1.map(item => item.programs_id)
```
Comment (jazzjacob) [inline app/companyCall.js:79]: T

File: components/subViews/company/matchList.vue [hunk 2]
```diff
@@ -1,36 +1,61 @@
...
+      <h3>
+        {{ post.title }}
+        {{ $t("looking-for") }}
+      </h3>
+      <div class="items">
+
+        <!-- Section: Bachelors -->
+        <div class="section">
+          <p class="section-heading">{{ $t("programs") }}</p>
+          <div class="tag-list">
```
Comment (jazzjacob) [inline components/subViews/company/matchList.vue:12]: shouldn't this be a `<ul>` with `<li>` inside?

File: components/subViews/company/matchList.vue [hunk 3]
```diff
@@ -1,36 +1,61 @@
...
-            <div v-if="post.qualifications" class="match-qualifications">
-              <p style="font-weight: 700">{{ $t("qualifications") }}</p>
-              <p>{{ post.qualifications[locale] }}</p>
+          </div>
+        </div>
+
+         <!-- Section: Masters -->
+        <div class="section">
+          <p class="section-heading">{{ $t("master_programs") }}</p>
+          <div class="tag-list">
```
Comment (jazzjacob) [inline components/subViews/company/matchList.vue:26]: shouldn't this be a `<ul>` with `<li>` inside?

File: components/subViews/company/matchList.vue [hunk 4]
```diff
@@ -1,36 +1,61 @@
...
+        <!-- Section: Qualifications -->
+        <div v-if="post.qualifications" class="section">
+          <p class="section-heading">{{ $t("qualifications") }}</p>
+          <p class="border-box">{{ post.qualifications[locale] }}</p>
+        </div>
+
+        <!-- Section: Positions -->
+        <div class="section">
+          <p class="section-heading">{{ $t("positions") }}</p>
+          <div class="tag-list">
```
Comment (jazzjacob) [inline components/subViews/company/matchList.vue:45]: shouldn't this be a `<ul>` with `<li>` inside?

**Review by jazzjacob — CHANGES_REQUESTED** [review]
Looks good, maybe minor HTML changes could be done. take look at my comments

File: components/subViews/company/matchList.vue (hunk 2 above)
Comment (olehoef) [reply components/subViews/company/matchList.vue:12]: maybe the tag-list. Is that very important tho? What difference does this make?

Comment (jazzjacob) [reply components/subViews/company/matchList.vue:12]: Exactly. It's just HTML best practice, nothing major - but better for accessibility, clarity etc.

Comment (olehoef) [reply components/subViews/company/matchList.vue:12]: ok will have a look

**Review by jazzjacob — DISMISSED** [review]

**Review by jazzjacob — APPROVED** [review]

**Pull request merged**